nautobot_ssot/contrib.py

```diff
@@ -257,6 +252,16 @@ class NautobotModel(DiffSyncModel):
 
     _model: Model
 
+    @classmethod
```

Comment:
I _think_ this probably doesn't need to be a `classmethod`?

Reply:
After the refactor I still have this as a `@classmethod`. We can meet and discuss.

Reply:
Nevermind, I now understand why this has to be like this.